styles/components/_hero.scss

```diff
@@ -0,0 +1,47 @@
+.hero {
+  background: $tundora 100% / cover;
+  background-position: center;
```

Comment:
in `background: $tundora 100% / cover;` the 100% means `background-position: 100%`. If you want to use the shorthand, then include other background properties in it.